fluent-plugin-google-cloud.gemspec

```diff
@@ -24,6 +24,7 @@ eos
   gem.add_runtime_dependency 'google-api-client', '~> 0.9.0'
   gem.add_runtime_dependency 'google-cloud-logging', '~> 0.23.2'
   gem.add_runtime_dependency 'googleauth', '~> 0.4'
+  gem.add_runtime_dependency 'google-protobuf', '~> 3.2.0.2'
```

Comment:
Seems `grpc` has it own dependency (https://github.com/grpc/grpc/blob/41373d0457a41801297e5fdd0ed03bfe94d26ff2/grpc.gemspec#L30)

`s.add_dependency 'google-protobuf', '~> 3.1'`

Would that affect us?